pkg/storage/unified/sql/data/resource_history_list_since_modified.sql

```diff
@@ -0,0 +1,18 @@
+SELECT
+    {{.Ident "guid"}},
+    {{.Ident "resource_version"}},
+    {{.Ident "namespace"}},
+    {{.Ident "group"}},
+    {{.Ident "resource"}},
+    {{.Ident "name"}},
+    {{.Ident "folder"}},
+    {{.Ident "value"}},
+    {{.Ident "action"}}
+FROM resource_history
+WHERE {{.Ident "namespace" }} = {{.Arg .Namespace }}
+  AND {{.Ident "group" }} = {{.Arg .Group }}
+  AND {{.Ident "resource" }} = {{.Arg .Resource }}
+  AND {{.Ident "resource_version" }} > {{.Arg .SinceRv }} -- needs to be exclusive of the sinceRv
```

Comment:
I'm assuming `SinceRV` is the current rv of the index. So we want all events exclusive of that.